libs/geospatial/src/main/java/org/codice/ddf/libs/location/CountryCodeConverter.java

```diff
@@ -0,0 +1,105 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.libs.location;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.MissingResourceException;
+import java.util.Objects;
+
+import org.apache.commons.lang.StringUtils;
+
+import ddf.security.PropertiesLoader;
+
+/**
+ * Utility to convert country codes into various formats.
+ */
+public class CountryCodeConverter {
+
+    private static final String FIPS_TO_ISO3 = "FIPS10-4toISO3.properties";
+
+    public static final String ENGLISH_LANG = "en";
+
+    private static Map<String, String> countryCodesMap;
+
+    /**
+     * Converts FIPS 10-4 into ISO 3166-1 alpha 3.
+     *
+     * @param fipsCountryCode a FIPS 10-4 country code
+     * @return an ISO 3166-1 alpha-3 country code
+     */
+    public static String convertFipsToIso3(String fipsCountryCode) {
+        String iso3CountryCode = getCountryCodesMap().get(fipsCountryCode);
+        if (StringUtils.isBlank(iso3CountryCode)) {
+            throw new MissingResourceException(
+                    "Could not find a corresponding ISO 3166-1 alpha 3 country code given the FIPS"
+                            + " 10-4 country code: " + fipsCountryCode,
+                    CountryCodeConverter.class.toString(),
+                    fipsCountryCode);
+        }
+        return iso3CountryCode;
+    }
+
+    /**
+     * Converts ISO 3166-1 alpha 3 into FIPS 10-4.
+     * <p>
+     * Special case:
+     * ISO 3166-1 alpha 3 country: "PSE" corresponds to FIPS 10-4 countries: "WE" and "GZ",
+     * in this case, the returned list will contain those two countries. In all other cases,
+     * the returned list will contain one country.
+     *
+     * @param iso3alphaCountryCode a FIPS 10-4 country code
+     * @return an ISO 3166-1 country code in a List<String>
+     */
+    public static List<String> convertIso3ToFips(String iso3alphaCountryCode) {
+        if ("PSE".equals(iso3alphaCountryCode)) {
+            return Arrays.asList("WE", "GZ");
+        }
+        String fipsCountryCode = getCountryCodesMap().entrySet()
+                .stream()
+                .filter(entry -> Objects.equals(entry.getValue(), iso3alphaCountryCode))
+                .map(Map.Entry::getKey)
+                .findFirst()
+                .orElse(null);
+        if (StringUtils.isBlank(fipsCountryCode)) {
+            throw new MissingResourceException(
```

Comment:
Fixed to return an empty list instead on an exception. Also removed the thrown exception in the convertFipsToIso3 method, it will now return null if there is no valid conversion.